### ISSUE TITLE
A case for a latency-tuned performance profile

### ISSUE THREAD

**Issue by ayoungethan** [pop-os/system76-power#95]
This is a follow up to my comment here: https://github.com/pop-os/system76-power/issues/2#issuecomment-501543903

I came across some interesting performance discussions regarding Mac OS tuning (from the engineers) and RedHat tuning. Here are the two PDFs: 

https://access.redhat.com/sites/default/files/attachments/201501-perf-brief-low-latency-tuning-rhel7-v1.1.pdf (see in particular p. 8 about performance profiles: "Because tuning for throughput often at odds with tuning for latency, profiles have been split along those boundaries as well providing a “balanced” profile").

Mac OS X it seems was tuned historically around audio performance and then battery life, because that is what revived the company and its core customer demographic. So OS X prioritizes audio latency above all other non-critical primary threads at all times, at the expense of throughput:
https://www.cse.unsw.edu.au/~cs9242/10/lectures/09-OSXAudio.pdf

Based on this, it seems to be somewhat of a case of "throughout, battery life, low latency: pick any two" and that philosophy could guide the shaping of performance profiles.

This means that anyone who needs a low latency system (such as for multimedia production) could benefit from a performance profile tuned to reduce both (average) latency and jitter (variation in latency).

In summary, I think it would be a mistake to conclude that a latency-tuned performance profile is only relevant and useful for certain server setups. It absolutely has a use in a desktop system with regards to multimedia responsiveness.
